src/App.css

```diff
@@ -1,38 +1,101 @@
-.App {
```

Comment:
I want to see you implement styled components into this, as you have all the css code it will be simple to put in :)